lib/js/components/Modals/Modal/Modal.vue

```diff
@@ -0,0 +1,417 @@
+<template>
+	<div
+		class="modal"
+		:class="{
+			'-danger': danger,
```

Comment:
do we need thia class?

Reply:
Yes, for prop `danger`. What can we do is to move it deeper, to `modal__content` div.

Reply:
I cannot find it. Can you point me to the line where we change styles based on this class?

Reply:
Ok, its redundant :)